checks/config_db.go

```diff
@@ -26,38 +21,25 @@ func (c *ConfigdbChecker) Run(ctx *canaryContext.Context) pkg.Results {
 	for _, conf := range ctx.Canary.Spec.ConfigDB {
 		results = append(results, c.Check(ctx, conf)...)
 	}
+
 	return results
 }
 
 func (c *ConfigdbChecker) Check(ctx *canaryContext.Context, extConfig external.Check) pkg.Results {
 	check := extConfig.(v1.ConfigDBCheck)
 	result := pkg.Success(check, ctx.Canary)
+
+	logger.Tracef("query: %v", check.Query)
+
 	var results pkg.Results
 	results = append(results, result)
-	logger.Tracef("[%v] query: %v", check.Host, check.Query)
-	var endpoint string
-	var query = url.QueryEscape(check.Query)
-	if strings.HasSuffix(check.Host, "/") {
-		endpoint = fmt.Sprintf("%v?query=%v", check.Host, query)
-	} else {
-		endpoint = fmt.Sprintf("%v/?query=%v", check.Host, query)
-	}
-	resp, err := http.Get(endpoint)
-	if err != nil {
-		return results.Failf("Failed to send GET request, %v", err)
-	}
-	defer resp.Body.Close()
-	if resp.StatusCode != http.StatusAccepted && resp.StatusCode != http.StatusCreated {
-		return results.Failf("Failed to get response, %v", resp.Status)
-	}
-	body, err := io.ReadAll(resp.Body)
+
+	res, err := duty.Query(ctx, db.Pool, check.Query)
 	if err != nil {
-		return results.Failf("Failed to read the response body: %v", err)
-	}
-	queryResult := ConfigDBQueryResult{}
-	if err := json.Unmarshal(body, &queryResult); err != nil {
-		results.Failf("failed to unmarshal the response body into QueryResult: %v", err)
+		return results.Failf("failed running query: %v", err)
 	}
+
+	queryResult := ConfigDBQueryResult{Results: res}
```

Comment:
For `ConfigDBQueryResult`, I haven't populated `Count` & `Columns` because the `duty.Query` only returns results. I can make that change to `Query` func if that's necessary.